core/src/main/scala/kafka/coordinator/group/GroupMetadata.scala

```diff
@@ -244,6 +244,25 @@ private[group] class GroupMetadata(val groupId: String, initialState: GroupState
       leaderId = members.keys.headOption
   }
 
+  def maybeElectNewLeader() {
+    leaderId match {
```

Comment:
You can use `foreach` instead of `match` if there is nothing to do in the `None` case